### PR TITLE
JAMES-2734 Document Sieve and ManageSieve

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: server/apps/distributed-app/docs/modules/ROOT/pages/configure/sieve.adoc [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+To be correctly executed, please note that the *Sieve* mailet is required to be positionned prior the
+*LocalDelivery* mailet.
+
+== Managing Sieve scripts
+
+A user willing to manage his Sieve scripts on the server can do so via several means:
+
+He can ask an admin to upload his script via the xref:operate/cli.adoc[CLI]
+
+As James support ManageSieve (link:https://datatracker.ietf.org/doc/html/rfc5804[RFC-5804]) a user
```
Comment (Arsnael) [inline server/apps/distributed-app/docs/modules/ROOT/pages/configure/sieve.adoc:31]:
```suggestion
As James supports ManageSieve (link:https://datatracker.ietf.org/doc/html/rfc5804[RFC-5804]) a user
```

File: src/site/xdoc/server/config-sieve.xml [hunk 2]
```diff
@@ -25,10 +25,76 @@
...
+
+      <p>To be correctly executed, please note that the <b>Sieve</b> mailet is required to be positionned prior the
+          <b>LocalDelivery</b> mailet.</p>
   </section>
+    <section name="Managing Sieve scripts">
+        <p>A user willing to manage his Sieve scripts on the server can do so via several means:</p>
+
+        <p>He can ask an admin to upload his script via the <a href="manage-cli.html">CLI</a></p>
+
+        <p>As James support ManageSieve (<a href="https://datatracker.ietf.org/doc/html/rfc5804">RFC-5804</a>) a user
```
Comment (Arsnael) [inline src/site/xdoc/server/config-sieve.xml:56]:
```suggestion
        <p>As James supports ManageSieve (<a href="https://datatracker.ietf.org/doc/html/rfc5804">RFC-5804</a>) a user
```

File: server/apps/distributed-app/docs/modules/ROOT/pages/configure/sieve.adoc [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+.imapserver.xml content
+|===
+| Property name | explanation
+
+| bind
+| Configure this to bind to a specific inetaddress. This is an optional integer value.  This value is the port on which this IMAP4 server is configured
+to listen. If the tag or value is absent then the service
+will bind to all network interfaces for the machine If the tag or value is omitted, the value will default to the standard IMAP4 port
+port 143 is the well-known/IANA registered port for IMAP
+port 993 is the well-known/IANA registered port for IMAPS  ie over SSL/TLS
```
Comment (Arsnael) [inline server/apps/distributed-app/docs/modules/ROOT/pages/configure/sieve.adoc:61]: You will have a formatting issue here I think

File: server/apps/distributed-app/docs/modules/ROOT/pages/configure/sieve.adoc [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+The standard children of the managesieveserver tag are:
+
+.imapserver.xml content
+|===
+| Property name | explanation
+
+| bind
+| Configure this to bind to a specific inetaddress. This is an optional integer value.  This value is the port on which this IMAP4 server is configured
+to listen. If the tag or value is absent then the service
+will bind to all network interfaces for the machine If the tag or value is omitted, the value will default to the standard IMAP4 port
```
Comment (Arsnael) [inline server/apps/distributed-app/docs/modules/ROOT/pages/configure/sieve.adoc:59]:
```suggestion
will bind to all network interfaces for the machine. If the tag or value is omitted, the value will default to the standard IMAP4 port
```

File: src/site/xdoc/server/config-sieve.xml [hunk 5]
```diff
@@ -25,10 +25,76 @@
...
+        <p>This tag has an optional boolean attribute - <strong>enabled</strong> - that defines whether the service is active or not.  The value defaults to "false" if
+            not present.</p>
+
+        <p>The standard children of the managesieveserver tag are:</p>
 
+        <dl>
+            <dt><strong>bind</strong></dt>
+            <dd>Configure this to bind to a specific inetaddress. This is an optional integer value.  This value is the port on which this ManageSieve server is configured
+                to listen. If the tag or value is absent then the service
+                will bind to all network interfaces for the machine If the tag or value is omitted, the value will default to the standard ManageSieve port
```
Comment (Arsnael) [inline src/site/xdoc/server/config-sieve.xml:80]:
```suggestion
                will bind to all network interfaces for the machine. If the tag or value is omitted, the value will default to the standard ManageSieve port:<br/>
```

File: server/apps/distributed-app/docs/modules/ROOT/pages/configure/sieve.adoc [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+all the relevant configuration for the ManageSieve server.  The behavior of the ManageSieve service is
+controlled by the attributes and children of this tag.
+
+This tag has an optional boolean attribute - *enabled* - that defines whether the service is active or not.
+The value defaults to "false" if
+not present.
+
+The standard children of the managesieveserver tag are:
+
+.imapserver.xml content
```
Comment (chibenwa) [inline server/apps/distributed-app/docs/modules/ROOT/pages/configure/sieve.adoc:52]:
```suggestion
.managesieveserver.xml content
```

File: server/apps/distributed-app/docs/modules/ROOT/pages/configure/sieve.adoc (hunk 3 above)
Comment (chibenwa) [inline server/apps/distributed-app/docs/modules/ROOT/pages/configure/sieve.adoc:61]:
```suggestion
| bind
| Configure this to bind to a specific inetaddress. This is an optional integer value.  This value is the port on which this ManageSieve server is configured to listen. If the tag or value is absent then the service
will bind to all network interfaces for the machine If the tag or value is omitted, the value will default to the standard ManageSieve port (port 4190 is the well-known/IANA registered port for ManageSieve.)
```

**Review by Arsnael — APPROVED** [review]

**Pull request merged**